### PR TITLE
Patrick day1

### PR DESCRIPTION
**Pull request by pwkurth**
Added day 1 problem from AoC (Node.js)

### CODE REVIEW

File: pwkurth/node/day_1/problem1.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//         }
+//         if (!previousDepth) {
+//             previousDepth = currentDepthSum
+//         } else {
+//             console.log("Here")
+//         }
+//         console.log(currentDepthSum, currentDepthWindow, previousDepth)
+//     }
+//     return total_increase
+// }
```
Comment (RookieOne) [inline pwkurth/node/day_1/problem1.js]: Remove :)

File: pwkurth/node/day_1/problem1.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+export const readPuzzleInput = filepath => {
+    const input = fs.readFileSync(filepath).toString().split("\n")
+
+    const mapped = input.map( x => parseInt(x));
+
+    return mapped
+
```
Comment (RookieOne) [inline pwkurth/node/day_1/problem1.js]: Get rid of blank line

File: pwkurth/node/day_1/problem1_case.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        it("Return the answer from the input", () => {
+            const input = readPuzzleInput("./src/day1/puzzle_reports.txt")
+
+            const answer = groupIncrease(input)
+
+            assert.equal(answer, 1262)
+        })
+        describe("Day 1 - Part 2", () => {
+            describe("Problem 1 - Part 2", () => {
+                it.only("Should be 5", () =>{
```
Comment (RookieOne) [inline pwkurth/node/day_1/problem1_case.js]: Remove only

File: pwkurth/node/day_1/problem1_part2.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        }else{
+        
+        //We get total of groupDepth (But only for A)
+        groupDepthTotal.push(groupDepth[0] + groupDepth[1] + groupDepth[2])
+
+        //Now we need to shift the index around somehow
+        groupDepth[0] = groupDepth[1]
+        groupDepth[1] = groupDepth[2]
+        groupDepth[2] = currentDepth
+
```
Comment (RookieOne) [inline pwkurth/node/day_1/problem1_part2.js:41]: Remove blank line

**Review by RookieOne — APPROVED** [review]
Left comments but also approved. :) Fix in future branch or with this branch.

File: pwkurth/node/day_1/problem1.js (hunk 2 above)
Comment (pwkurth) [reply pwkurth/node/day_1/problem1.js]: No.

**Pull request merged**